app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild

```diff
@@ -28,8 +28,11 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-7.1-lastshort.patch
 	"${FILESDIR}"/${PN}-7.1-bluray_srm+pow.patch
 	"${FILESDIR}"/${PN}-7.1-bluray_pow_freespace.patch
+	"${FILESDIR}"/${PN}-7.1-clang.patch
 )
 
+DOCS=( index.html )
```

Comment:
Are there no other docs included? Wouldn't this be better to be installed as `HTML_DOCS`?

Reply:
This is the only doc.  And it's a text file, not html.  Just has a weird name